azure-boards-common/src/main/java/com/synopsys/integration/azure/boards/common/model/AzureSpecTemplate.java

```diff
@@ -0,0 +1,44 @@
+package com.synopsys.integration.azure.boards.common.model;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import org.apache.commons.lang3.StringUtils;
+
+public class AzureSpecTemplate {
```

Comment:
This seems like it would be well suited in our int-rest library. A utility to handle this kind of replacement for urls.  Might be something to look into in the future.  I like the concept here.